Kickstarter-iOS/Views/Controllers/RewardsCollectionViewControllerTests.swift

```diff
@@ -118,9 +128,126 @@ final class RewardsCollectionViewControllerTests: TestCase {
       |> Project.lens.state .~ .live
       |> Project.lens.rewardData.rewards .~ [reward]
 
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.features .~ [
+        OptimizelyFeature.rewardLocalPickupEnabled.rawValue:
+          true
+      ]
+
     combos(Language.allLanguages, [Device.phone5_8inch]).forEach {
       language, device in
-      withEnvironment(language: language, locale: .init(identifier: language.rawValue)) {
+      withEnvironment(
+        language: language,
+        locale: .init(identifier: language.rawValue),
+        optimizelyClient: optimizelyClient
+      ) {
+        let vc = RewardsCollectionViewController.instantiate(
+          with: project,
+          refTag: nil,
+          context: .createPledge
+        )
+        _ = traitControllers(device: device, orientation: .portrait, child: vc)
+
+        FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)")
+      }
+    }
+  }
+
+  func testRewards_LocalPickUp_FeatureDisabled_RewardNotBacked_NoLocalPickupRewardShown_Success() {
+    let reward = Reward.template
+      |> Reward.lens.shipping.preference .~ .local
+      |> Reward.lens.localPickup .~ .canada
+
+    let project = Project.cosmicSurgery
+      |> Project.lens.state .~ .live
+      |> Project.lens.rewardData.rewards .~ [.noReward, reward]
+
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.features .~ [
+        OptimizelyFeature.rewardLocalPickupEnabled.rawValue:
+          false
+      ]
+
+    combos(Language.allLanguages, [Device.phone5_8inch]).forEach {
+      language, device in
+      withEnvironment(
+        language: language,
+        locale: .init(identifier: language.rawValue),
+        optimizelyClient: optimizelyClient
+      ) {
+        let vc = RewardsCollectionViewController.instantiate(
+          with: project,
+          refTag: nil,
+          context: .createPledge
+        )
+        _ = traitControllers(device: device, orientation: .portrait, child: vc)
+
+        FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)")
```

Comment:
Is the function `FBSnapshotVerifyView` the verification portion of these tests? Does it either return true or raise an error?

Reply:
It's part of our snapshot testing framework `github "uber/ios-snapshot-test-case" == 6.2.0` which is a dependency in our `Cartfile`. (Carthage is what we use to manage third party dependency alongside Swift Package Manager).

Reply:
Its' just a function that creates snapshots and stores them in the `Screenshots` folder with the `identifier` listed.